boxoffice/templates/boxoffice.js

```diff
@@ -254,7 +254,7 @@ $(function() {
                 }
               });
 
-              if(valid_discount_coupon) {
+              if (valid_discount_coupon) {
                 boxoffice.ractive.set('order.line_items',line_items);
```

Comment:
Is this `set` required? Setting `pre_applied` on the `discount_policy` should re-render the UI?